File: swagger/models/get_staff_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .staff import Staff

class GetStaffResponse(BaseModel):
	"""
	no description available

	Attributes:

		staff_members
		A list of staff members.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'StaffMembers': ['staff_members', Staff],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'staff_members': 'StaffMembers',
		}

	pagination_response = None
	staff_members = None

File: swagger/Models/GetStaffResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetStaffResponse extends BaseModel
{
	/**
	 * no description available
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * A list of staff members.
	 * 
	 * @var Staff[] 
	 */
	public $staffMembers;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'StaffMembers' => ['staffMembers', 'array', Staff::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'staffMembers' => ['StaffMembers', 'array', Staff::class],
			];
	}
}
